prow/pjutil/trigger_test.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pjutil
+
+type fakeJobResult struct {
+	err error
```

Comment:
Remainder?

Reply:
We removed all the functions which we were testing in the earlier iteration so I had to remove the corresponding tests as well. I'll add the tests for the new functions, just want to confirm that the overall code looks good before that.

Reply:
This is still unreachable code